.github/workflows/main.yml

```diff
@@ -150,6 +150,6 @@ jobs:
         uses: ncipollo/release-action@v1
         with:
           name: ${{ env.PACKAGE_STRING }}
-          bodyFile: CHANGELOG.md
+          bodyFile: RELEASE_NOTES.md
```

Comment:
Why?

Reply:
There is no `CHANGELOG.md` file in the Chocolate Doom repository, but there is one in Crispy Doom. I renamed it to `RELEASE_NOTES.md` for clarity. Perhaps I should squash these commits.

Reply:
There is a `ChangeLog` (without `.md`). Should either use that directly or convert it to markdown.

Reply:
This is the content of `ChangeLog`:
```
Chocolate Doom's version history is stored in Git. For a full
change log, clone the repository on Github:

    https://github.com/chocolate-doom/chocolate-doom
```

We have `NEWS.md`, but it's too long and too verbose. I think the point of `RELEASE_NOTES.md` is to have the most important news for regular users to read.

Reply:
Is there a way to automatically cut `NEWS.md` so it's reasonable?

Reply:
Perhaps, but we can just copy and paste it into `RELEASE_NOTES.md` and maybe trim it down a bit.